GameDefinition/config.ui.default

```diff
@@ -15,6 +15,18 @@
 [additional_commands]
 ; Angband compatibility (accept target)
 Cmd_2 = ("KP_Insert", ([CmdAim], "", ByAimMode AimModeCmd {exploration = ExecuteIfClear Dashboard, aiming = Accept}))
+; Custom key rebinding example:
+;
+;   Cmd_3 = ("a", ([], "My Happy Cmd", Macro["t"]))
+;
+; Explaination: to (t)rigger a consumeable item for use, you normally use the key press 't'.
+; In this case, suppose you would like to use the key press 'a' instead of 't'. 
+;   Cmd_3          - can be anything as long as it doesn't conflict or overlap with something
+;   "a"            - the new key to use 
+;   "[]"           - can either  [] or [CmdAim]. If using [CmdAim] your keybinding will be displayed from the help menu, and you would even be able to activate it from there
```

Comment:
be

Reply:
Also, `CmdItem` probably matches it better, since that's the category used for "t".

Might I also ask for 80-character lines? :)

Reply:
No problem 👍